### PR TITLE
docs(evaluate-artifacts): refactor into 2 pags DOC-347

### PR DESCRIPTION
**Pull request by 317brian**
Resolves Jira: [DOC-347]

[DOC-347]: https://armory.atlassian.net/browse/DOC-347

### CODE REVIEW

**Comment by 317brian** [comment]
@stephenatwell 
admin/operator page (the current hidden page will redirect here): https://deploy-preview-536--armory-docs.netlify.app/docs/armory-admin/evaluate-artifacts-stage-enable/
pipeline maker page: https://deploy-preview-536--armory-docs.netlify.app/docs/spinnaker-user-guides/evaluate-artifacts-stage-use/

File: content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  Enable the Evaluate Artifacts Stage plugin to add a stage to Armory Enterprise that makes it easy to evaluate SpEL expressions inside of Spinnaker™ artifacts.
+aliases:
+  - /docs/plugin-guide/plugin-evaluate-artifacts/
+---
+
+![Proprietary](/images/proprietary.svg)
+
+## Overview
+
+Armory’s Evaluate Artifacts plugin allows you to easily evaluate SpEL queries inside of any Spinnaker artifacts. Some artifact types and their associated stages, such as Kubernetes, support  using SpEL to read parameters from the Spinnaker context and inject them into an artifacts manifest at deploy time. Other stages, such as the Terraform Integration stage, do not support this out of the box. The Evaluate Artifacts Stage plugin adds supports for evaluating SpEL against any artifact within your Spinnaker pipeline.
```
Comment (bbrina) [inline content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md]: There's a double space after `Kubernetes, support  `

File: content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+For information about how to use the stage, see [Use the Evaluate Artifacts Stage]({{< ref "evaluate-artifacts-plugin-use.md" >}}).
+
+## Setup
```
Comment (bbrina) [inline content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md:23]: Question: should we add Halyard setup too?

File: content/en/docs/spinnaker-user-guides/evaluate-artifacts-plugin-use.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+profile = "prod"
+```
+
+
+
+## Known issues
+
+### Problem saving artifacts
+
+You may run into an issue where it seems like artifacts (or changes to them) are not being saved even though you click **Save Changes**. This issue occurs because of how the UI handles updates to artifacts in relation to changes to other configurations.
```
Comment (bbrina) [inline content/en/docs/spinnaker-user-guides/evaluate-artifacts-plugin-use.md]: Breaks my heart 💔

**Review by bbrina — COMMENTED** [review]
TL;DR should we add Halyard setup?

File: content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md (hunk 2 above)
Comment (stephenatwell) [reply content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md:23]: yeah..... we should.

File: content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+Add the following snippet to your Operator config, such as `spinnakerservice.yml`:
```
Comment (aimeeu) [inline content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md]: @brian-armory `spinnakerservice.yml` isn't the Operator config. It's the Spinnaker manifest.  I clarified the terminology in the Operator docs PR.

File: content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+                  enabled: true
+                  plugins:
+                    Armory.EvaluateArtifactsPlugin:
+                      enabled: true
+                      version: <PLUGIN_VERSION> # Replace with the version you want to use. Omit the 'v' when providing a version. For example, use 0.1.0, not v0.1.0
+{{< /prism >}}
+
+Keep the following in mind when using this configuration snippet: 
+
+* Make sure to include the nested `spinnaker` parameters on lines 4 and 5. Both are required because of how Operator configs are processed and consumed by Armory Enterprise.
```
Comment (aimeeu) [inline content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md]:
```suggestion
* Make sure to include the nested `spinnaker` parameters on lines 4 and 5. Both are required because of how Spinnaker consumes plugin configuration.
```

File: content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
+---
+title: Enable the Evaluate Artifacts Stage Plugin
+description: >
+  Enable the Evaluate Artifacts Stage plugin to add a stage to Armory Enterprise that makes it easy to evaluate SpEL expressions inside of Spinnaker™ artifacts.
+aliases:
+  - /docs/plugin-guide/plugin-evaluate-artifacts/
+---
+
+![Proprietary](/images/proprietary.svg)
```
Comment (aimeeu) [inline content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md:9]: Does this mean the plugin only works with Armory Enterprise?

File: content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+## Overview
+
+Armory’s Evaluate Artifacts plugin allows you to easily evaluate SpEL queries inside of any Spinnaker artifacts. Some artifact types and their associated stages, such as Kubernetes, support  using SpEL to read parameters from the Spinnaker context and inject them into an artifacts manifest at deploy time. Other stages, such as the Terraform Integration stage, do not support this out of the box. The Evaluate Artifacts Stage plugin adds supports for evaluating SpEL against any artifact within your Spinnaker pipeline.
+
+For information about how to use the stage, see [Use the Evaluate Artifacts Stage]({{< ref "evaluate-artifacts-plugin-use.md" >}}).
+
```
Comment (aimeeu) [inline content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md:22]: Before `Setup`, what do you think of including a _Before you begin_  section that points to the OSS plugin use guide https://spinnaker.io/guides/user/plugins/ ? The first few sections cover terminology, but the latter half details how to install a plugin using Halyard.

File: content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+Keep the following in mind when using this configuration snippet: 
+
+* Make sure to include the nested `spinnaker` parameters on lines 4 and 5. Both are required because of how Operator configs are processed and consumed by Armory Enterprise.
+* Replace `<PLUGIN_VERSION>` on lines 10 and 22 with the version of the plugin that you want to use. Plugin versions can be found [here](#versions).
+
+Then, deploy your updated Operator configuration using one of the following methods:
+
+- The `deploy.sh` script if you use the [Armory kustomize repo](https://github.com/armory/spinnaker-kustomize-patches) to help manage your Operator configurations
+- `kubectl -n <spinnaker-namespace> -f <operator-config>.yml​`
```
Comment (aimeeu) [inline content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md]: You are deploying an Armory Enterprise manifest, not an updated Operator configuration. 

My understanding of `deploy.sh` is that it's for a quickstart deployment only, that you wouldn't use it for deploying an updated Armory Enterprise manifest. How you deploy Spinnaker or AE depends on whether you are using a single `spinnakerservice.yml` manifest file or are using Kustomize patches.  

If using a single manifest file:
```bash
kubectl -n <namespace> apply -f <path-to-manifest-file>
```

If using Kustomize patches:
```bash
kubctl -n <namespace> apply -k <path-to-kustomize-directory>
```
or if you have Kustomize installed:

```bash
# the namespace is declared in kustomization.yml
# run this from the same directory as your kustomization.yml file
kustomize build | kubectl  apply -f -
```

**Review by aimeeu — CHANGES_REQUESTED** [review]

File: content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md (hunk 7 above)
Comment (317brian) [reply content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md:22]: I have the directions for Halyard written up but haven't pushed them yet. But I'll add the link in the pre-reqs so people can reference the terminology.

File: content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md (hunk 6 above)
Comment (317brian) [reply content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md:9]: It means it's a proprietary plugin and is not open source

**Comment by 317brian** [comment]
Halyard steps added and tested

File: content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md (hunk 6 above)
Comment (aimeeu) [reply content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md:9]: Ok. Stephen's blog post says it works with AE and Spinnaker, so seeing the Proprietary graphic made me think it only works with AE.

File: content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+  - /docs/plugin-guide/plugin-evaluate-artifacts/
+---
+
+![Proprietary](/images/proprietary.svg)
+
+## Overview
+
+Some artifact types and their associated stages, such as Kubernetes, support  using SpEL to read parameters from the Spinnaker context and inject them into an artifacts manifest at deploy time. Other stages, such as the Terraform Integration stage, do not support this out of the box. The Evaluate Artifacts Stage plugin adds supports for evaluating SpEL against any artifact within your Spinnaker pipeline.
+
+For information about how to use the stage, see [Use the Evaluate Artifacts Stage]({{< ref "evaluate-artifacts-stage-use.md" >}}).
```
Comment (aimeeu) [inline content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md]:
```suggestion
For information about how to use the stage, see {{< linkWithTitle "evaluate-artifacts-stage-use.md" >}}.
```

You can use the `linkWithTitle` shortcode if you don't want to duplicate the page title in a normal link

**Review by aimeeu — APPROVED** [review]

File: content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+
+3. Run `hal deploy apply` to apply the changes to your Armory Enterprise instance.
+
+{{% /tab %}}
+{{< /tabs>}}
+
+## Versions
+
+- 0.1.0 - Improved the user experience. Execution errors for the stage now display in the UI.
+- 0.0.10 - Initial Release
```
Comment (aimeeu) [inline content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md:130]:
```suggestion
- 0.0.10 - Initial Release

## {{% heading "nextSteps" %}}

* {{< linkWithTitle "evaluate-artifacts-stage-use.md" >}}
```

Comment (aimeeu) [reply content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md:130]: This type of linking helps with SEO also
